packages/shared-components/src/formio/components/core/accordion/Accordion.tsx

```diff
@@ -10,6 +10,7 @@ class Accordion extends BaseComponent {
       label: 'Trekkspill',
       type: 'accordion',
       key: 'accordion',
+      hideLabel: true,
```

Comment:
Er dette nødvendig for oversettelser, eller er denne endringen ikke relevant for denne PRen?